pkg/storage/image.go

```diff
@@ -555,6 +556,13 @@ func (svc *imageService) ResolveNames(imageName string) ([]string, error) {
 	}
 	domain, remainder := splitDockerDomain(imageName)
 	if domain != "" {
+		registry, err := sysregistriesv2.FindRegistry(&types.SystemContext{}, domain)
```

Comment:
(You can use `nil` instead of `&types.SystemContext{}`.)

Is it consistent to use an empty context here, while `GetImageService` makes it a variable?

(IIRC the only caller calls `GetImageService` with `nil`, so it may not matter.)